pkgs/development/interpreters/ueforth/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, stdenv, fetchFromGitHub
+, nodejs, python
+}:
```

Comment:
```suggestion
{ lib, stdenv, fetchFromGitHub, nodejs, python }:
```

Reply:
I like to separate library functions to facilitate the build from actual build inputs in this way, but ok, I guess for few parameters like this, it can fit on a single line ...